sdks/java/extensions/sorter/README.md

```diff
@@ -0,0 +1,44 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+Sorter
+============
+
+The provided Sorter can be used to sort large (more than can fit to memory)
```

Comment:
Changed README as recommended. The original version only had one sorter (what is now called ExternalSorter) and left it as public, though the user only had to manipulate an Options object for it. Changed README to not go into detail on the implementations and made implementations other than BufferedExternalSorter package private. Leaving that one public for options specification.